arangod/Aql/FixedVarExpressionContext.cpp

```diff
@@ -62,7 +62,8 @@ void FixedVarExpressionContext::serializeAllVariables(velocypack::Options const&
   for (auto const& it : _vars) {
     builder.openArray();
     it.first->toVelocyPack(builder);
-    it.second.toVelocyPack(&opts, builder, true);
+    it.second.toVelocyPack(&opts, builder, /*resolveExternals*/false,
```

Comment:
this is now calling `toVelocyPack` with `resolveExternals = false`, whereas previously it was called with `resolveExternals = true`. Is that an intentional change?

Reply:
Any comment on this?